starknet_py/tests/e2e/fixtures/accounts.py

```diff
@@ -235,3 +237,48 @@ async def argent_account_v040(
 
     await deploy_result.wait_for_acceptance()
     return deploy_result.account
+
+
+@pytest_asyncio.fixture(scope="package")
+async def eth_account(
+    eth_fee_contract: Contract,
+    strk_fee_contract: Contract,
+    client,
+    eth_account_class_hash,
+) -> BaseAccount:
+    private_key = _get_random_private_key_unsafe()
+
+    signer = EthSigner(private_key, chain_id=StarknetChainId.SEPOLIA)
+
+    # Manually serialize u512 into felt array
+    serializer = Uint256Serializer()
+    public_key_bytes = signer.public_key.to_bytes(64, byteorder="big")
+    constructor_calldata = serializer.serialize(
+        int.from_bytes(public_key_bytes[:32], byteorder="big")
+    ) + serializer.serialize(int.from_bytes(public_key_bytes[32:], byteorder="big"))
+
+    address, salt = _new_address(eth_account_class_hash, constructor_calldata)
+
+    await prepay_account(
+        address=address,
+        eth_fee_contract=eth_fee_contract,
+        strk_fee_contract=strk_fee_contract,
+    )
+
+    account = Account(
+        address=address,
+        client=client,
+        signer=signer,
+        chain=StarknetChainId.SEPOLIA.value,
```

Comment:
nit: No need to access `.value`
```suggestion
        chain=StarknetChainId.SEPOLIA,
```

```python
print(StarknetChainId.SEPOLIA)
print(StarknetChainId.SEPOLIA.value)
```

gives

```
393402133025997798000961
393402133025997798000961
```